Makefile

```diff
@@ -60,7 +60,7 @@ test-coverage: katana-genesis load-env
 benchmark-madara:
 	cd benchmarks && bun i && bun run benchmark-madara
 
-test-target: load-env
+test-target: katana-genesis load-env
```

Comment:
can we remove this? test-target is more for local testing and adding the `katana-genesis` rule to it makes it impractical to use.

Reply:
no won't work, but also don't want to rerun it each time